### PR TITLE
Adds PIDSubsystem and TrapezoidProfileSubsystem to Commands2.

### PR DESCRIPTION
**Pull request by lospugs**
…/robotpy-commands-v2#28

### CODE REVIEW

File: commands2/pidsubsystem.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+class PIDSubsystem(Subsystem):
+    def __init__(self, controller: PIDController, initial_position: float = 0.0):
+        """
+        Creates a new PIDSubsystem.
+
+        :param controller: The PIDController to use.
+        :param initial_position: The initial setpoint of the subsystem.
+        """
+        self.m_controller: PIDController = controller
```
Comment (virtuald) [inline commands2/pidsubsystem.py:18]: type annotation not needed

_controller not m_controller

File: commands2/pidsubsystem.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        self.setSetpoint(initial_position)
+        self.addChild("PID Controller", self.m_controller)
+        self.m_enabled = False
```
Comment (virtuald) [inline commands2/pidsubsystem.py:21]: Use `_enabled` instead of `m_enabled.

**Review by virtuald — COMMENTED** [review]
Formatting and mypy failed

**Comment by lospugs** [comment]
Is the overall preference that I squash the commits into one and push again?

**Comment by virtuald** [comment]
Yes, I like commits to tell a useful history most of the time.

**Pull request merged**